randovania/exporter/hints/joke_hints.py

```diff
@@ -5,37 +5,11 @@
     # 1. They should clearly be jokes, and not real hints or the result of a bug.
     # 2. They shouldn't reference real-world people.
     # 3. They should be understandable by as many people as possible.
-    #
+    # These hints are used if a game does not have joke hints defined.
     "By this point in your run, you should have consumed at least 200 mL of water to maintain optimum hydration.",
-    "Make sure to collect an Energy Transfer Module; otherwise your run won't be valid!",
-    "Adam has not yet authorized the use of this hint.",
     "Back in my day, we didn't need hints!",
-    "Hear the words of O-Lir, last Sentinel of the Fortress Temple. May they serve you well.",
-    "Warning! Dark Aether's atmosphere is dangerous! Energized Safe Zones don't last forever!",
-    "A really important item can be found at - (transmission ends)",
-    "Did you know that Bigfoot and Santa Claus exist in the Metroid Prime canon?",
-    "I hear. Them. Everywhere. They're coming. Can't sleep. Ever. They'll eat me. Eat.",
-    "Space Pirates, strangely, dislike theft.",
-    "Power Bomb Expansions are just space hamburgers.",
-    "While in Morph Ball mode, press &image=SI,0.70,0.68,D523DE3B; to drop Bombs.",
-    "Charge your beam to fire a normal shot when out of ammo.",
-    "Movement in Morph Ball mode is faster than unmorphed, even without Boost Ball.",
-    "While walking, holding L makes you move faster.",
     "I wonder how many players will see this message...?",
```

Comment:
Connsidering its origin i wonder if itd make more sense for this one to get moved too.
Don't mind it either way though.

Reply:
I wasn't sure on this one at first either, but it's generic enough to the point where I think it's ok.